src/instructlab/utils.py

```diff
@@ -389,7 +389,7 @@ def read_taxonomy_file(file_path: str, yaml_rules: Optional[str] = None):
                 f"{file_path_p} is not valid. The top-level element is not an object with key-value pairs."
             )
             errors += 1
-            return None, warnings, errors
+            return warnings, errors
```

Comment:
(github interface is stupid so I leave the comment here) you missed the same change in line 386. (We should probably add a typing descriptor for the return value for the function; later.)

Reply:
Good catch!